### PR TITLE
Regenerated from API - made UserSignup.Spec.IdentityClaims required

### PR DESCRIPTION
**Pull request by sbryzak**
API PR: https://github.com/codeready-toolchain/api/pull/400

### CODE REVIEW

**Review by rajivnathan — COMMENTED** [review]
Looking at the CRD yaml it looks like only these are required.

```
required:
- email
- preferredUsername
- sub
```
probably because the others are marked as optional https://github.com/codeready-toolchain/api/blob/1e69b9614bd81720b4de6d405f5438c3331f4182/api/v1alpha1/usersignup_types.go#L191-L201

I guess you need to remove the `+optional` from the individual fields within `identityClaims` to make them all required.

**Review by alexeykazakov — COMMENTED** [review]
It looks like this PR is based on some outdated commit.

File: deploy/registration-service/registration-service.yaml [hunk 1]
```diff
@@ -114,7 +114,7 @@ objects:
                 successThreshold: 1
                 timeoutSeconds: 1
               readinessProbe:
-                failureThreshold: 30
```
[the remark this answers is not in the document]
Comment (sbryzak) [reply deploy/registration-service/registration-service.yaml]: Not intended, my branch wasn't up to date.  Fixed.

File: config/crd/bases/toolchain.dev.openshift.com_usersignups.yaml [hunk 2]
```diff
@@ -131,6 +131,11 @@ spec:
                 description: The cluster in which the user is provisioned in If not
                   set then the target cluster will be picked automatically
                 type: string
+<<<<<<< HEAD
```
Comment (alexeykazakov) [inline config/crd/bases/toolchain.dev.openshift.com_usersignups.yaml:134]: This needs some cleanup after resolving a conflict :)

**Review by alexeykazakov — APPROVED** [review]

File: config/crd/bases/toolchain.dev.openshift.com_usersignups.yaml (hunk 2 above)
Comment (sbryzak) [reply config/crd/bases/toolchain.dev.openshift.com_usersignups.yaml]: Fixed

**Review by ranakan19 — APPROVED** [review]

**Review by rajivnathan — APPROVED** [review]

**Pull request merged**